server/requirements.txt

```diff
@@ -12,7 +12,7 @@ flask-restful>=0.3.9
 flask-sqlalchemy
 gunicorn
 humanize
-pquisby
+pquisby==0.0.17
```

Comment:
I understand the need, but are you sure you want to include this change in this PR?  (It would be more appropriate to do this in its own PR, for reasons which I can provide if you can't come up with adequate ones on your own.)

And, you like `==0.017` better than a "less-than" or a list of working releases?

Reply:
I suppose it's useless to hope for a fix today, but my thought at the moment was that I wanted a clean test run and it could be a temporary hack.

Although I'm starting to think that maybe we really want to lock quisby in particular to the specific version we think we need; and, for right now, to leave it locked to 0.0.17 until we have a new version that provides features/fixes we need. But I don't see any point in trying to be fancier than "the" known version. Although if you do, I'm happy to have provoked that discussion here before I think about attempting a "permanent" strategy.

Reply:
> we really want to lock quisby in particular to the specific version we think we need; and, for right now, to leave it locked to 0.0.17 until we have a new version that provides features/fixes we need.

I am thinking the exact same thing.  (Once burned, twice shy; twice burned....)

> I don't see any point in trying to be fancier than "the" known version.

I was just thinking about how the `b0.73` branch got broken because it wanted  '14 (although we didn't know that) and it got '17, and so having our requirement be "one of these N versions which we know will work" would be better (except that it wouldn't have helped `b0.73`...).

So, yes, I think we should pin to the version which satisfies our requirements (at least for Quisby...).

Reply:
> I think we should pin to the version which satisfies our requirements

Now, whether we want to do that in this PR or not depends on whether, in the future, we are going to want to cherry-pick any of the commits that this PR merges separately from any of the other commits that come with them.  (Like, we probably want to pin Quisby in the `b0.73` branch, and possibly in the `b0.72` branch...and I don't know whether we want to pull the rest of these changes along with that.)

Reply:
> we probably want to pin Quisby in the `b0.73` branch

Well, yeah, but it's already pinned at `<0.0.13`....

Reply:
Right; and b0.72 doesn't even reference quisby, because that's much older. So we're really only worrying about `main`.

I'm not against a separate PR, and I threw this in out of simple frustration because I wanted to get a CI run. On the other hand, we really don't need to worry about separating the version lock from the rest so I don't think it's a big deal except in the Jira view.

Reply:
> we're really only worrying about `main`.

🎊  😅 

> I threw this in out of simple frustration because I wanted to get a CI run.

Understood.

> we really don't need to worry about separating the version lock from the rest so I don't think it's a big deal except in the Jira view.

I'm not clear on what the Jira angle is, but I think it would be nice to be able to look at the sequence of commits in the log and to be able to see where we put in (or last updated...)  the rev-lock for Quisby.  (But, I suppose, I can get to that info via "blame" instead of looking at the log....)